dist/generator-vulcanjs/lib/ui-text.js

```diff
@@ -22,6 +23,7 @@ const descriptions = {
 
 const messages = {
   appName: 'App name',
+  doShallowClone: 'Use fast (<5s) or complete (>60s) git clone? See unshallow action for more "fast" details.',
```

Comment:
Same issue here. Don't you think we might be confusing people here? @SachaG what do you think?